api.php

```diff
@@ -41,6 +41,7 @@
 require_once('lib/gamemessage.php');
 require_once('board/orders/jsonBoardData.php');
 require_once('variants/install.php');
+// require_once('./vendor/aws/aws-autoloader.php');
```

Comment:
This is leftover from me trying to load a PHP aws library. We can remove whenever, it doesn't do any harm. I quickly realized that doing this in nodejs would be much faster since all the tooling libraries are available for javascript.